File: Practica1/Ejercicio4/src/practica1ejercicio4/Cliente.java
package practica1ejercicio4;

public class Cliente {
    private AdminFiltros admin;
    
    public Cliente(AdminFiltros admin){
        this.admin = admin;
    }
    
    public void enviarCorreo(String email, String password){
        Correo correo = new Correo(email, password);
        admin.postCorreo(correo);
    }
}

File: Practica2/ejercicio_grupal1/lib/Cliente.dart
import 'AdminFiltros.dart';
import 'Correo.dart';

class Cliente {
  final AdminFiltros admin;

  Cliente(this.admin);

  MapEntry<bool,String> enviarCorreo(String email, String password) {
    Correo correo = Correo(email, password);
    return admin.postCorreo(correo);
  }
}